app/external_version_code.txt

```diff
@@ -1 +1 @@
-2014150878
\ No newline at end of file
+2014150879
```

Comment:
Take the version number, from firebase, CI will increase it by 1
![Screenshot 2023-08-09 at 9 56 15 AM](https://github.com/kickstarter/android-oss/assets/4083656/46673ee5-424c-4e75-acc2-3c48faa806bd)